src/Math/Measures.dfy

```diff
@@ -26,14 +26,22 @@ module Measures {
     f(i) + CountableSum(f, i+1)
   }
 
-  ghost const boolSampleSpace: iset<bool> := iset _: bool
+  ghost function Powerset<A(!new)>(): iset<A> {
```

Comment:
This is not really the powerset though. isn't it? It's just the infinite set of all elements in A. Imo, the powerset is returned by DiscreteSigmaAlgebra below.